app/repositories/auth.repository.ts

```diff
@@ -60,7 +60,6 @@ export const createUser = async (data: {
       .then(rows => rows[0]);
 
     await trx('accounts').insert({
-      id: uuidv4(),
```

Comment:
Why was this removed?

Reply:
I did not remove it. I just adjusted it

Reply:
Where?

Reply:
const result = await trx('users')
      .insert({
        id: uuidv4(),
        username: data.username,
        email: data.email,
        first_name: data.firstName,
        last_name: data.lastName,
        password: data.password,
      })
      .returning('*')
      .then(rows => rows[0]);

    await trx('accounts').insert({
      user_id: result.id,
      balance: 0.0,
      is_default: true,
    });

    await trx.commit();
    return result;

I refactored it since it was repetitive, result.id is already from the id generated